domains/htaamas.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "HTaamas",
+    "email": "zero@htaamas.xyz",
+    "discord": "htaamas"
+  },
+  "records": {
+    "URL": "htaamas.xyz"
```

Comment:
URL records should have protocols (HTTP or HTTPS). Re-open PR when this is fixed.